src/__tests__/lib/utils/SynapseClient.test.tsx

```diff
@@ -104,6 +105,24 @@ it('get files', () => {
     })
 })
 
+it ('should attempt to add files to download list and return an error for anonimous user', () => {
```

Comment:
@jay-hodgson , since we are calling the live endpoint I could not think of a way to generate/pass token so the only response I could capture is the error when trying to upload as an anonymous user. Do you have a tip on how to test a happy path?

Reply:
1.  This error message is likely to change in the future (backend should block anonymous user from adding any files to their DL list. But I could clear the anonymous user DL list today and the "max of 100 files" error will not occur!).
2.  SynapseClient.test is full of integration level tests, rather than unit tests.  And I'm not convinced that integration level tests should be part of the automated build process!  We could leave these to manual testing of the client, or add a special node script that could be manually run.
3.  I think "fail(reason)" is cleaner than "expect(false).toBe(true)"